src/QtLocationPlugin/MapProvider.h

```diff
@@ -40,16 +40,19 @@ class QNetworkRequest;
 
 class MapProvider : public QObject
```

Comment:
Did you have a reason to keep this?
```suggestion
class MapProvider
```

Reply:
Only reason being I forgot to remove it